.appveyor.yml

```diff
@@ -1,5 +1,17 @@
 # Documentation: https://wiki.unvanquished.net/wiki/Continuous_integration
 
+only_commits:
+  files:
+    - .appveyor.yml
+    - appveyor/
+    # The rest of this list should stay in sync with azure-pipelines.yml
+    - src/
+    - libs/
+    - cmake/
+    - CMakeLists.txt
+    - '*.cmake'
+    - .gitmodules
```

Comment:
Some stuff in `tools/` is used by the build.

Reply:
I have not added it on purpose because I planned to do that:

- https://github.com/DaemonEngine/Daemon/pull/1135

The remaining things in `tools/` is not built by the CI (it's arm64 specific).